### DISCUSSION TITLE
Division of micro texture region

### DISCUSSION THREAD

**Discussion by Ming-is** [neperfepx/neper#417] [Q&A] [answered]
The overall domain is a rectangular prism of length 1.5 mm in both the x and z directions, and 1 mm in the y direction .Microtextured regions are demarcated via – planes at 0.5 and 1.0mm offsets from the origin in the  direction. 
![8} QZUTX}NJPJIL~N%EDW 0](https://user-images.githubusercontent.com/71123582/160223768-a889323f-c515-4005-86ab-4756f93d0e98.png)
Rve as shown in the above figure, the grains in the light color area are assigned random orientation, and the micro texture area represented by the dark color area is assigned custom direction.
So what instructions should I use in neper？

**Comment by rquey** [comment, marked as answer]
Generate a tessellation in the full domain, export the cell positions using `-statcell x,y,z`, define the cell orientations based on the cell positions, and import them back using `-ori file(...)`.
